code/game/machinery/deployable.dm

```diff
@@ -271,24 +274,46 @@
 	icon = 'icons/obj/dropwall.dmi'
 	icon_state = "dropwall"
 	item_state = "grenade"
-	mode = NORTH
+	mode = AUTO
 	var/uptime = DROPWALL_UPTIME SECONDS
```

Comment:
Could remove this var and use the define directly since we are doing "MINUTES" there.

Reply:
Could but it depends if we want to change it again